src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/HelperTests/LanguageHelperTest.cs

```diff
@@ -14,11 +14,11 @@ public class LanguageHelperTest
         [InlineData("12", false)]
         [InlineData("", false)]
         [InlineData(null, false)]
-        [InlineData("xx", false)]
-        [InlineData("tv", false)]
-        public void IsTwoLetterISONameTest(string language, bool expected)
+        [InlineData("norsk", false)]
+        [InlineData("t1", false)]
+        public void IsTwoLetterTest(string language, bool expected)
         {
-            bool result = LanguageHelper.IsTwoLetterISOName(language);
+            bool result = LanguageHelper.IsTwoLetters(language);
             Assert.Equal(result, expected);
```

Comment:
A minor detail, but you have switched expected and actual.

Reply:
Thanks, fixed.